packages/ui/src/components/ui/attachment/attachmentLanding/attachmentLanding.tsx

```
@@ -0,0 +1,85 @@
+        <div className="flex flex-row flex-flex-wrap items-center justify-between">
+          <div className="flex flex-row gap-2 items-center">
+            <AnimatePresence>
+              {files.length ? (
+                <ScrollArea>
+                  <div className="flex flex-row gap-2">
+                    {files.map((file) => (
+                      <motion.div
+                        exit={{ opacity: 0, scale: 1 }}
+                        initial={{ opacity: 0, scale: 0.7 }}
+                        animate={{ opacity: 1, scale: 1 }}
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)